tests/load/pipeline/utils.py

```diff
@@ -160,7 +160,7 @@ def _drop_dataset_sql(schema_name: str) -> None:
                 for schema_name in p.schema_names:
                     drop_func(schema_name)
 
-        p._wipe_working_folder()
+        # p._wipe_working_folder()
```

Comment:
Is this intentional?

Reply:
no :) but also not harmful, I'll do separate commit to fix